src/50_Samples_%26_Templates/Star_Rating.html

```diff
@@ -0,0 +1,141 @@
+<!--
+  #### Introduction
+
+  This star rating widget is implemented using only CSS, given AMP's restriction on custom JavaScript. It still has the key features of star rating components:
+
+   * touch, mouse and keyboard accessibility
+   * stars change color when the user mouses over them
+   * once a selection is made, it "sticks"
+   * clean scalable vector icons
+   * screen reader friendly
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/samples_templates/star_rating/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <!-- #### Setup
```

Comment:
nit: can you please place the import below the boilerplate and put the setup headline in a separate comment. This results in a better formatting in the sample view as both imports will be shown and no unrelevant code (this makes use of how the parser works to improve the formatting - I'd be happy to make this more explicit, but haven't got an idea how).

```
  <!-- #### Setup -->
  <!-- 
    Import the `amp-form` and `amp-mustache` components.
  -->
  <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>

  <!--
    #### CSS for the star rating
  -->
```